toolkit/scripts/pkggen.mk

```diff
@@ -67,30 +75,17 @@ $(graph_file): $(specs_file) $(go-grapher)
 		$(logging_command) \
 		--output $@
 
+# We want to detect changes in the RPM cache, but we are not responsible for directly rebuilding any missing files.
+$(CACHED_RPMS_DIR)/%: ;
+
 # Remove any packages which don't need to be built, and flag any for rebuild if
 # their dependencies are updated.
 ifneq ($(CONFIG_FILE),)
 # If an optional config file is passed, validate it and any files it includes. The target should always depend
 # on the value of $(CONFIG_FILE) however, so keep $(depend_CONFIG_FILE) always.
 # Actual validation is handled in imggen.mk
-$(optimized_file): $(validate-pkggen-config)
+$(cached_file): $(validate-pkggen-config)
```

Comment:
>$(cached_file): [](start = 0, length = 15)

It looks like me way have this target defined twice now in this file. Won't `make` complain?

Reply:
Makefile will merge the two targets, joining any dependency targets

Reply:
Wow, interesting, thank you.

---
In reply to: [499724770](https://github.com/microsoft/CBL-Mariner/pull/141#discussion_r499724770) [](ancestors = 499724770)